packages/react/src/components/AccountSettings/DeleteUser/DeleteUser.tsx

```diff
@@ -26,6 +23,12 @@ function DeleteUser({
 
   const { user, isLoading } = useAuth();
 
+  // subcomponents
+  const { Error, SubmitButton, Warning } = React.useMemo(
+    () => ({ ...DEFAULTS, ...(components || {}) }),
```

Comment:
Nittish, prefer nullish coalescing

Reply:
Done in [45ca1e2](https://github.com/aws-amplify/amplify-ui/pull/3123/commits/45ca1e2d34ac9160af0dcfa8439a7d7bc611d2f7) for both `DeleteUser` and `ChangePassword`